contracts/SampleRecipient.sol

```diff
@@ -15,9 +16,10 @@ contract SampleRecipient is RelayRecipient {
         get_relay_hub().deposit.value(msg.value)();
     }
 
-    function withdraw() public {
+    function withdraw() public onlyOwner {
```

Comment:
for this `onlyOwner` we copy in contract from openZepellin. I don't like it.

Reply:
fixed